client/src/livingdocs-component-app/app.js

```diff
@@ -87,11 +87,11 @@ export class App {
     if (this.selectedItemIndex !== undefined) {
       const item = await getItem(this.selectedItems[this.selectedItemIndex].id);
       this.title = item.conf.title;
+      await this.loadDisplayOptions();
```

Comment:
Do we really have to `await` this before loading the preview?
If not I would run let the preview load before we finished loading the displayOptions, they can just appear in the UI later on when they have finished loading.

Reply:
Yes the await is needed. Because the displayOptions are needed before the preview is loaded. Otherwise the the preview will show the title even though the displayOptions have `hideTitle: true` set per default.

Reply:
ah i see. makes sense. thanks for explaining.